Formula/tj.rb

```diff
@@ -0,0 +1,21 @@
+class Tj < Formula
+  desc "Line timestamping tool for stdin"
+  homepage "https://github.com/sgreben/tj"
+  url "https://github.com/sgreben/tj/archive/6.0.2.tar.gz"
+  sha256 "c667f8fb09839a677bd6b3d6c06103bce59ffb2e33ec1bd7b49abfc00f02f5dd"
+  head "https://github.com/sgreben/tj.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/sgreben/tj").install buildpath.children
+    cd "src/github.com/sgreben/tj" do
+      system "make", "binaries/osx_x86_64/tj"
+      bin.install "binaries/osx_x86_64/tj"
+    end
+  end
+  test do
+    pipe_output("#{bin}/tj", "test")
```

Comment:
Can we check that the exit code is zero, and match some expected content in the output?